api/nodes/serializers.py

```diff
@@ -124,9 +128,124 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NodeContributorsSerializer(JSONAPISerializer):
+    """ Separate from UserSerializer due to necessity to override almost every field as read only
+    """
+    filterable_fields = frozenset([
+        'fullname',
+        'given_name',
+        'middle_name',
+        'family_name',
+        'id',
+        'bibliographic',
+        'permissions'
+    ])
+    id = ser.CharField(source='_id', label='ID')
```

Comment:
Oh, I see. The client sends the ID when adding a contributor; disregard previous comment.